src/main/java/net/praqma/tracey/broker/impl/filters/PayloadJSONFilter.java

```diff
@@ -15,6 +15,9 @@
 public class PayloadJSONFilter implements TraceyFilter {
 
     private String pattern = "";
+    private String key = "";
```

Comment:
I see that you declared two fields as private final in TraceyRabbitMQReceiverImpl class but here you don't use final. Why? If there is no good reason not to use final then we should add it